bindings/core/src/method/secret_manager.rs

```diff
@@ -19,7 +19,22 @@ use crate::OmittedDebug;
 #[serde(tag = "name", content = "data", rename_all = "camelCase")]
 #[non_exhaustive]
 pub enum SecretManagerMethod {
-    /// Generate Ed25519 addresses.
+    // TODO: Should we refactor `GetAddressesOptions` so we can use it for this method as well?
```

Comment:
Do we have an issue for this or in which PR is this supposed to resolved?

Reply:
wanted to discuss this first, no issue yet

Reply:
Bit weird to add the TODOs then I'd say, I think it would be better to first discuss and then add the TODO or open an issue that just references it without any TODO

Reply:
don't see the problem ... we did this so many times now ... if in the discussion we think it's worth an issue, then one is created and the comment is replaced with the issue number.

Reply:
With discussion you mean gh discussion?